chrome-extension/content.css

```diff
@@ -19,3 +20,33 @@
   padding: 5px;
   display: none;
 }
+#lobster-submit-button, #newCategoryButton {
```

Comment:
Did you change all of these to id to override styling or for a different reason?

Reply:
yes, was intended to override styling